rpki-validator/src/main/resources/messages.properties

```diff
@@ -76,6 +76,10 @@ repository.object.maximum.size.error=RPKI object {0} size {1} is greater than th
 repository.object.is.trust.anchor.certificate.error=RPKI object {0} is not a trust anchor
 repository.object.is.trust.anchor.certificate.passed=RPKI object {0} is a trust anchor
 
+manifest.crl.next.update.time.matches.error=Manifest next update time {0} does not match CRL next update time {1}
```

Comment:
Do we need to indicate that match here means within 60 seconds?

Reply:
I'm not sure it is very valuable and in the future this time will be reduced (maybe to 0).